### ISSUE TITLE
Sample `variables.gradle` file?

### ISSUE THREAD

**Issue by jcmorrow** [Meso-Health/meso-enrollment#1]
Hi folks! I'm having some trouble getting the project to build because of missing variables in the build settings. To be precise, I'm seeing this message upon opening the project in Android Studio:
```
ERROR: Cannot get property 'rollbarApiKey' on extra properties extension as it does not exist
```
I assume that that variable is probably set in `variables.gradle` as referenced in the README:
>  In order to run the application, you also need a variables.gradle file in your root directory which stores environment variables. Create a file called variables.gradle and copy the file contents from 1Password.

Could you provide a sample `variables.gradle` that will not expose any Watsi secrets but will provide enough dummy values for the app to build?

**Comment by jbeorse** [comment]
Ah, this is a failure of our README. You can actually just rename `variable.gradles.tmpl` to `variables.gradle` and it should work fine, at least for debug builds. It doesn't need a valid Rollbar API key, just to have something exist in the `variables.gradle` file.

I'll update the README and then close the issue. Let me know if that doesn't work for you.

**Comment by liangmicha** [comment]
Hi @jcmorrow , thanks for the issue! 

This [PR](https://github.com/Meso-Health/meso-enrollment/pull/2) contains some updates, feel free to copy contents of `variables.gradle.tmpl` from there. If you're setting up a local build against the server running locally, shouldn't need to set anything more.

**Comment by jcmorrow** [comment]
Thank you!

**Comment by jbeorse** [comment]
Resolved in #2
